components/accelerator/nvidia/infiniband/component_output.go

```diff
@@ -94,56 +93,58 @@ func ParseStatesToOutput(states ...components.State) (*Output, error) {
 	return nil, errors.New("no state found")
 }
 
+var (
+	msgNoAtLeastPortsOrRateSet = "no at least ports or at least rate set"
```

Comment:
the English is hard to read. change it to `must set ports or rate`. What does "at least" mean? can add it to the message if needed